app/main.py

```diff
@@ -1,8 +1,50 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
```

Comment:
style is still not fixed :) 
tests are failed because of that

make sure flake check is passed